Upload files to MVS datasets

kristinochka opened the pull request:
![MVS_Upload](https://user-images.githubusercontent.com/14590297/59229851-9a8a4e80-8ba9-11e9-8d3f-ce08d53da44a.gif)

If file name is too long, it's automatically truncated.

kristinochka commented:
@ChrisBoehmCA Could you take a look?

kristinochka commented:
Connected to this GitHub issue: https://github.com/zowe/vscode-extension-for-zowe/issues/86

kristinochka commented:
@Colin-Stone How about changing "Upload" to "Upload member"? Usually the menu specifies the object user is adding, for example  "New File or New Folder". User should understand where they uploading to because they clicked on a dataset.

Colin-Stone commented:
> @Colin-Stone How about changing "Upload" to "Upload member"? Usually the menu specifies the object user is adding, for example "New File or New Folder". User should understand where they uploading to because they clicked on a dataset.
Works for me.. I would still like the .. to represent a subsequent dialog

kristinochka commented:
> Upload seems a little ambiguous. Maybe "Upload to.." would be better with the dots indicating that a dialog follows?.
> Seem to have lost the registerCommand check nr12. (zowe.removeSession) I intend to rewrite this piece soon because it is a pain to maintain.

Changed the string to "Upload member..", also added the missing unit test for nr12. I agree, this section needs to be re-factored. Every time a new action is added all other 48 actions need to be updated.

Not sure if I'll have time today to add an integration test. I can also follow up with it in another PR.

kristinochka commented on __tests__/mvs/mvsNodeActions.test.ts:
```diff
@@ -0,0 +1,33 @@
+
+const testTree = DatasetTree();
+
+describe("mvsNodeActions", async () => {
```
Done.
